packages/core/networks/hardhat/deployed.json

```diff
@@ -0,0 +1,5765 @@
+{
```

Comment:
A few questions:
1. Is this pulled in by hardhat scripts or does that use the generated files under artifacts/?
2. Are the abis needed? If not, is there an easy way to drop them from this file to make it easier to read/interpret?

Reply:
This is not used in any hardhat scripts. This is generated by `yarn hardhat export --export-all /path/to/deployed.json`, which is run manually by the caller. This is so we can programmatically save deployed addresses from the local `deployments/` directory which does not get committed.

The ABI's are not needed, but `export` doesn't strip them out of the box.

Reply:
The main use case of `export` i think is to allow a front-end client to run the `export` command from CLI and easily pull in abi/addresses into their app. So maybe just don't commit this file for now